administrator/components/com_content/views/article/view.html.php

```diff
@@ -55,7 +56,8 @@ public function display($tpl = null)
 		if (count($errors = $this->get('Errors')))
 		{
 			JError::raiseError(500, implode("\n", $errors));
-			return false;
+			
+            return false;
```

Comment:
@nikitadhiman why dont you set up PHPCS. Instructions here - 
https://docs.joomla.org/Joomla_CodeSniffer